### ISSUE TITLE
[206] false positie

### ISSUE THREAD

**Issue by sblaisot** [warpnet/salt-lint#313]
**Describe the bug**

Rule 206 gives false positive whenever if finds `}}` even if it's inside a jinja block

```
$ salt-lint init.sls 
[206] Jinja variables should have spaces before and after: '{{ var_name }}'
init.sls:83
  {%- do hash_variable.update({'tls': {'cert': '/path/to/cert.pem', 'key': '/path/to/cert.key'}}) %}
```

**To Reproduce**

Add the above line that extends a hash variable in your sls file

**Expected behavior**

No trigger for rule 206

**Comment by fatsolko** [comment]
same
```
[206] Jinja variables should have spaces before and after: '{{ var_name }}'
kafka2/map.connector.jinja:25
"group_users": pillar.system_users | default({grains.id:{'groups':[]}}),
```
